packages/twenty-utils/congratulate-dangerfile.ts

```diff
@@ -0,0 +1,109 @@
+import { danger } from 'danger';
+
+const ordinalSuffix = (number) => {
+  const v = number % 100;
+  if (v === 11 || v === 12 || v === 13) {
+    return number + 'th';
+  }
+  const suffixes = { 1: 'st', 2: 'nd', 3: 'rd' };
+  return number + (suffixes[v % 10] || 'th');
+};
+
+const fetchContributorStats = async (username: string) => {
+  const apiUrl = `https://twenty.com/api/contributors/contributorStats/${username}`;
+
+  const response = await fetch(apiUrl);
+  const data = await response.json();
+  return data;
+};
+
+const fetchContributorImage = async (username: string) => {
+  const apiUrl = `https://twenty.com/api/contributors/${username}/og.png`;
+
+  await fetch(apiUrl);
+};
+
+const getTeamMembers = async () => {
+  const org = 'twentyhq';
+  const team_slug = 'core-team';
+  const response = await danger.github.api.teams.listMembersInOrg({
+    org,
+    team_slug,
+  });
+  return response.data.map((user) => user.login);
+};
+
+const runCongratulate = async () => {
+  const pullRequest = danger.github.pr;
+  const userName = pullRequest.user.login;
+
+  const staticExcludedUsers = [
```

Comment:
In a follow-up PR could you please cleanup the code and make sure we have one source of truth for that list of users? We don't want to have to modify it in different places